guide/how-it-works/ecash/fedimint.md

```diff
@@ -0,0 +1,177 @@
+---
+
+<!--
+
+Editor's notes
+
+Explains generally how fedimint works
+
+Illustration sources
+
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/fedimint.jpg"
+   retina = "/assets/images/guide/how-it-works/ecash/fedimint@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/ecash/fedimint-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/fedimint-mobile@2x.jpg"
+   alt-text = "ecash overview illustration"
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Fedimint
+Fedimint is the first bitcoin-backed ecash protocol. Fedimint decentralizes trust across a federation of guardians, ensuring that no single guardian has complete control over user funds. Fedimint allows for various Federation configurations, each catering to different needs. These configurations include **Federated mints**, **Federated single guardian mints**, and **Solo mints (Unfederated mints)**, each with its strengths and trade-offs.
+
+## Role of guardians
+Guardians are participants who collaboratively manage a federations operations. They secure the federation's funds using multi-sig wallets, validate payments through consensus, and control ecash issuance and redemption.
+
+## Role of gateways
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/gateways.png"
+   retina = "/assets/images/guide/how-it-works/ecash/gateways@2x.png"
+   mobile = "/assets/images/guide/how-it-works/ecash/gateways.png"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/gateways@2x.png"
+   alt-text = "Network diagram showing how gateways connect federations (represented by blue dots) with the lightning network (represented by orange dots). The nodes are linked by dashed lines, visualizing the interconnected structure of federations and lightning nodes."
+   caption = "Gateways connect a federation’s ecash and bitcoin via the lightning network."
+   width = 800
+   height = 508
+   layout = "full-width"
+%}
+
+A gateway is a service that facilitates interactions between the federation (which operates largely off-chain) and the lightning network. A gateway acts as a bridge, enabling users within the federation to make payments to and receive payments on the lightning network.
+
+**How gateways work**: The gateway accepts bitcoin payments on the lightning Network and converts them into the bitcoin-backed ecash tokens used within the federation. It can also convert bitcoin-backed ecash tokens into bitcoin and send them over the lightning network. This is crucial because it allows the users within a federation to interact with the outside bitcoin and lightning network.
+
+**Vetted gateways**: A vetted gateway is one that has been approved by the federation's guardians as reliable and trustworthy. This vetting process helps to ensure that users' payments are handled by gateways that have a track record of reliability, reducing the likelihood of payment failures.
```

Comment:
The concept of unvetted gateways is being removed in the LNv2 module. All gateways will be vetted, since although they can be trusted to not steal user funds, they still must be trusted to provide a good user experience by routing quickly and not maliciously locking up funds in-flight. LNv2 won't be used until fedimint v0.6 (at least 3 months out) so it might be fine to just remove this - use your own discretion.